packages/react-components/react-tabster/etc/react-tabster.api.md

```diff
@@ -68,8 +68,8 @@ export const useFocusFinders: () => {
     findAllFocusable: (container: HTMLElement, acceptCondition?: ((el: HTMLElement) => boolean) | undefined) => HTMLElement[];
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 5 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26372&commitId=a8ff2ec8a1c499b54ceb93ea4e05533b40383b05&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1255 screenshots unchanged, 0 screenshots with different dimensions and 5 screenshots with visible difference.


  <details><summary><b>unknown</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Dropdown Example Loading Teams High Contrast.chromium.png|222| Changed|
|Dropdown.Dropdown Example Teams Dark.Focuses first item.chromium.png|637| Changed|
|Dropdown.Dropdown Example Teams Dark.Focuses second item.chromium.png|637| Changed|
|Dropdown.Dropdown Example Teams Dark.Focuses trigger.chromium.png|596| Changed|
|Dropdown.Dropdown Example Teams Dark.chromium.png|641| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>